web/themes/custom/recipe_theme/src/sass/view/_user_account_page.scss

```diff
@@ -0,0 +1,53 @@
+.user__name,
+.user__field-first-name,
+.user__field-introduction {
+  .field__label {
+    text-align: center;
+    font-family: 'Dosis', sans-serif;
+  }
+}
+
+.user__name,
+.user__field-first-name {
+  .field__label,
+  .field__item {
+    font-style: normal;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: $gray-800;
+  }
+  .field__label {
+    margin-top: 0.5rem;
+  }
+  .field__item {
+    background-color: $gray-200;
+    padding: 1rem auto;
+    border-radius: 0.4rem;
+    font-weight: 300;
+    text-align: center;
+  }
+}
+
+.user__field-introduction {
+  .field__label {
+    margin-top: 1.5rem;
+    font-size: 1.875rem;
+    line-height: 3.125rem;
+  }
+  .field__item {
+    text-align: justify;
+    text-indent: 2rem;
+    font-family: 'Open Sans';
+  }
+}
+
+.user__user-picture {
+  display: flex;
```

Comment:
If the only purpose here is to center align an image then `text-align: center;` should be enough. Flex is overkill just for horizontally center aligning stuff.